### ISSUE TITLE
Import buttons not showing in Foundry v10

### ISSUE THREAD

**Issue by Kiberia** [spetzel2020/pc-importer#5]
Hey! Any chance this could be updated to v10?

Unfortunately, when I enable PC Importer plugin, Import buttons used for, erm, import are not visible.

I can provide any extra info if needed

**Comment by spetzel2020** [comment]
Definitely. I have Quick Encounters to finish fixing, and need to publish
an updated Compendium Browser.

If you don't see any activity on this by mid next week, feel free to.ping
here again.

**Comment by spetzel2020** [comment]
Reproduced in Foundry v10: I'm not sure what you mean by "Import buttons" are not visible. In my case, the "Person+" icon for PC Importer doesn't show up at all because of these load errors which are no longer accessible:
```
import Actor5e from "/systems/dnd5e/module/actor/entity.js";    //default export
import {DND5E} from "/systems/dnd5e/module/config.js";
```

**Comment by spetzel2020** [comment]
Commented out the imports in PC Importer v0.9.0 and replaced above definitions with direct Actor calls.

**Comment by spetzel2020** [comment]
Partially fixed in 0.9.0a. However Issue #4 means no additional detail is being loaded.

**Comment by Kiberia** [comment]
Hi, sorry I'm new to this, so please bear with me. Can you advise how to install 0.9.0a from manifest?

I reinstalled add-on from `https://github.com/spetzel2020/pc-importer/releases/latest/download/module.json`, Foundry v10. Still no Import buttons:

<img width="1114" alt="image" src="https://user-images.githubusercontent.com/35463968/191030507-0fe0ffbc-96a4-497c-b9ac-b08476f1dc6a.png">

**Comment by spetzel2020** [comment]
I haven't built a new release yet.

**Comment by BlackDragonBE** [comment]
@spetzel2020 Sorry to bug you, but do you have any updates or workarounds? Thanks!

**Comment by spetzel2020** [comment]
No further progress yet; have been fixing other modules. Hopefully next
week. Unfortunately it's not just code; I have no idea how to replicate
things from the previous version.

**Comment by Ronon444** [comment]
After waiting for a solution for this problem, I have found a different application that can do this. So for people coming to this thread hoping for a fix, you can us [this](https://blackdragonbe.itch.io/mpmb-to-foundryvtt) in the meantime.
 
So @Kiberia, if you are still looking for something, I would HIGHLY recommend [this](https://blackdragonbe.itch.io/mpmb-to-foundryvtt)!

**Comment by spetzel2020** [comment]
Awesome to see there's an alternative!

**Comment by rjbprime** [comment]
@spetzel2020 Is there any update on version 0.9 being pushed out? Looking to use a free solution before having to spend limited funds.

Kind regards.
